backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/v1/bpartner/bpartnercomposite/JsonRetrieverService.java

```diff
@@ -426,6 +426,11 @@ private static JsonResponseLocation toJson(@NonNull final BPartnerLocation locat
 					.billTo(locationType.getIsBillToOr(false))
 					.billToDefault(locationType.getIsBillToDefaultOr(false))
 					.changeInfo(jsonChangeInfo)
+					.setupPlaceNo(location.getSetupPlaceNo())
```

Comment:
what the hell is setupPlaceNo? :)

Reply:
I didn't check. The requirement was to display in this template all that is displayed in their custom bpartner window, location tab. I only excluded the generated "Address" , the org and the client.